aiocoap/protocol.py

```diff
@@ -1058,6 +1120,8 @@ def on_cancel(self, callback):
     def __repr__(self):
         return '<%s %s at %#x>'%(type(self).__name__, "(cancelled)" if self.cancelled else "(%s call-, %s errback(s))"%(len(self.callbacks), len(self.errbacks)), id(self))
 
+MulticastResponse = ClientObservation
```

Comment:
ClientObservation provides all I needed to implement multicast, and the fact that observer issues never happen doesn't impact operation.  The new name is just for readability.